src/app/transactions/fields/StakingPositionFormField.tsx

```diff
@@ -59,7 +53,9 @@ export function StakingPositionFormField({
                     "validatorAddress",
                     stakingPosition.validatorAddresses[0]
                   );
-                  //setDecimals(stakingPosition.decimals);
+
+                  // Trigger the prop passed to the component
```

Comment:
e.g this comment. Brings zero value as it explains nothing, only describes the obvious.
It's the equivalent of :
```
// Do one plus one
const i = 1 + 1;
```